lib/acts_as_taggable_on_mongoid/taggable/changeable.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
+  module Taggable
+    # Overides of methods from Mongoid::Changeable
+    module Changeable
```

Comment:
Created this new.

ActsAsTaggableOn uses ActiveRecord to inject the lists into the model as a value so that ActiveRecord manages the change functions.

I could add the list as a field, but then it would be persisted to the database.  So I instead added this Changeable module to manages changes.